doc/project/project_roles.rst

```diff
@@ -50,51 +49,41 @@ Contributors are granted the following rights and responsibilities:
-the `Zephyr devel mailing list <https://lists.zephyrproject.org/g/devel>`_ or on
-the `Zephyr Discord Server <https://chat.zephyrproject.org>`_.
-
-Contributors who show dedication and skill are granted the Triage permission
-level to the Zephyr GitHub repository.
-
-You may nominate yourself, or another GitHub user, for promotion to the Triage
-permission level by creating a GitHub issue, using the :github:`nomination
-template <new?assignees=&labels=Role+Nomination&template=006_nomination.yml>`.
-
-Contributors granted the Triage permission level are permitted to add reviewers
-to a pull request and can be added as a reviewer by other GitHub users.
-Contributor change requests or approval on pull requests are not counted with
-respect to accepting and merging a pull request. However, Contributors comments
-and requested changes should still be considered by the pull request author.
 
 Zephyr Contributor Badge
-++++++++++++++++++++++++
+------------------------
 
-When your first contribution to the Zephyr project gets merged, you'll become eligible to claim your
-Zephyr Contributor Badge. This digital badge can be displayed on your website, blog, social media
-profile, etc. It will allow you to showcase your involvement in the Zephyr project and help raise
-its awareness.
+When your first contribution to the Zephyr project gets merged, you'll become
+eligible to claim your Zephyr Contributor Badge. This digital badge can be
+displayed on your website, blog, social media profile, etc. It will allow you to
+showcase your involvement in the Zephyr project and help raise its awareness.
 
-You may apply for your Contributor Badge by filling out the `Zephyr Contributor Badge form`_.
+You may apply for your Contributor Badge by filling out the `Zephyr Contributor
+Badge form`_.
 
-.. _collaborator:
+   .. _collaborator:
 
 Collaborator
 ++++++++++++
 
-A *Collaborator* is a Contributor who is also responsible for the maintenance
-of Zephyr source code. Their opinions weigh more when decisions are made, in a
+A *Collaborator* is a Contributor who is also participating in the maintenance
+of Zephyr source code. Their input weighs more when decisions are made, in a
 fully meritocratic fashion.
 
-Collaborators have the following rights and responsibilities,
-in addition to those listed for Contributors:
+You become a collaborator by showing consistent involvement in the project,
+including making significant contributions to the project over a period of
+time and by participating in the development and review of at least one area.
 
-* Right to set goals for the short and medium terms for the project being
-  maintained, alongside the Maintainer.
+To request to become a Collaborator, submit a pull request adding yourself
+to the ``collaborators`` section of an area in the :ref:`maintainers_file` and
+notify the maintainers of that area. The addition needs to be approved by
+the maintainers of that area.
```

Comment:
```suggestion
notify the existing maintainers of that area. The addition needs to be approved by
the maintainers of that area.
```
Otherwise this sounds like they must approve it.